src/Aspire.ProjectTemplates/templates/aspire-starter/.template.config/template.json

```diff
@@ -49,6 +49,12 @@
           "exclude": [
             "Aspire-StarterApplication.1.Tests/**/*.*"
           ]
+        },
+        {
+          "condition": "(TestFramework != \"xUnit.net\")",
+          "exclude": [
+            "Aspire-StarterApplication.1.Tests/XunitLogger.cs"
```

Comment:
Where is this file coming from?

Reply:
ah it's no longer there, will update this